scripts/install.sh

```diff
@@ -713,7 +713,7 @@ elif [ ! -d $DIRECTORY ]; then
 fi;
 
 if [ $DIRECTORY != "-" ]; then
-  plugins=( 'OctoDash Companion' 'Display Layer Progress' 'Filament Manager' 'Spool Manager' 'Preheat Button' 'Enclosure' 'Print Time Genius' 'Ultimaker Format Package' 'PrusaSlicer Thumbnails' 'TPLinkSmartPlug' 'Tasmota' 'TasmotaMQTT', 'Ophom (Phillips HUE)')
+  plugins=( 'OctoDash Companion' 'Display Layer Progress' 'Filament Manager' 'Spool Manager' 'Preheat Button' 'Enclosure' 'Print Time Genius' 'Ultimaker Format Package' 'PrusaSlicer Thumbnails' 'TPLinkSmartPlug' 'Tasmota' 'TasmotaMQTT' 'Ophom (Phillips HUE)' 'Tuya')
```

Comment:
I'm not sure listing most of these makes sense honestly. Maybe just the companion plugin? Merely installing OctoDash doesn't suddenly mean a user has interest in smart plug support. But, may as well keep the list consistent for now

Reply:
It might be good to whittle down the list for installation in a future PR, yeah!